web/src/main/webapp/css/classic/main.css

```diff
@@ -11,7 +11,6 @@
 
 BODY {
 	font-family: "Trebuchet MS", Arial, Helvetica, sans-serif;
-	font-size: 75%;
```

Comment:
@MALfunction84 Are you ok with this L&F change?

![psi-probe](https://cloud.githubusercontent.com/assets/975267/13621923/05c340f2-e569-11e5-96f1-47fa11d02978.png)